style.css

```diff
@@ -0,0 +1,15 @@
+.container {
```

Comment:
the commit message does not reflect the actual task/work. Please update the commit message